docs/es/features/census_configuration.md

```diff
@@ -1,156 +1,143 @@
 
-1. Acceder a través del panel de administración de su aplicación a la sección **Configuración > Funcionalidades** y activar el módulo de **Configurar conexión al censo remoto (SOAP)** como se puede ver a continuación:
-  ![Feature enabled](../../img/remote_census/feature-enabled-es.png)
+La información a rellenar esta dividida en tres apartados:
 
-## Configuración
+### Información General**
 
-Una vez activada la funcionalidad, podremos acceder a la sección **Configuración > Configuración Global** y clicar en la pestaña **Configuración del Censo Remoto**.
-En esta pantalla se podrá rellenar toda la información necesaria para poder configurar la conexión con el Censo de cada Ayuntamiento.
+- **Endpoint**: Nombre del host donde se encuentra el servicio del Censo con el que queremos conectarnos (wsdl).
 
-La información a rellenar esta dividida en tres apartados:
+### Datos de la Petición
+
+En esta sección rellenaremos todos los campos necesarios para poder realizar una petición para verificar un usuario contra un censo.
 
-1. **Información General**
-  - **Endpoint**: Nombre del host donde se encuentra el servicio del Censo con el que queremos conectarnos (wsdl).
+Para ayudar a entender como rellenar cada uno de los campos, nos basaremos en un supuesto WebService que recibe un método llamado `:get_habita_datos` con la siguiente estructura:
 
-    ![General information - Endpoint](../../img/remote_census/general-information-endpoint-es.png)
+```ruby
+{
+  request: {
+    codigo_institucion: 12,        #Valor estático
+    codigo_portal:      5,         #Valor estático
+    codigo_usuario:     10,        #Valor estático
+    documento:          12345678Z, #Valor dinámico relacionado con Número de Documento
+    tipo_documento:     1,         #Valor dinámico relacionado con Tipo de Documento
+    codigo_idioma:      102,       #Valor estático
+    nivel:              3          #Valor estático
+  }
+}
+```
 
-1. **Información para realizar la Petición**
+Campos necesarios para la petición:
 
-  En esta sección rellenaremos todos los campos necesarios para poder realizar una petición para verificar un usuario contra el Censo del Ayuntamiento.
+- **Nombre del método de la petición**: Nombre del método que acepta el WebService del censo.
 
-  ![Request Data](../../img/remote_census/request-data-es.png)
+  ![Datos de la petición - Nombre del método de la petición se rellena con get_habita_datos](../../img/remote_census/request-data-method-name-es.png)
+- **Estructura de la petición**: Estructura de la petición que recibe el WebService del censo. Los valores _fijos_ de esta petición deberán informarse. Los valores _dinámicos_ relacionados con Tipo de Documento, Número de Documento, Fecha de Nacimiento y Código Postal deberán dejarse con valor _null_.
```

Comment:
"informarse" sounds a bit strange :thinking:. What do you think about something like:

```suggestion
- **Estructura de la petición**: Estructura de la petición que recibe el WebService del censo. Los valores _fijos_ de esta petición deberán indicarse. Los valores _dinámicos_ relacionados con Tipo de Documento, Número de Documento, Fecha de Nacimiento y Código Postal deberán dejarse con valor _null_.
```